src/components/DividerBlock/DividerBlock.tsx

```diff
@@ -0,0 +1,22 @@
+import BackHandOutlinedIcon from "@mui/icons-material/BackHandOutlined";
+import { Box, Divider, Typography } from "@mui/material";
+import { useTranslation } from "react-i18next";
+
+import styles from "./Divider.module.scss";
+
+export const DividerBlock = () => {
+  const { t } = useTranslation();
+
+  return (
+    <div className={styles.divider}>
```

Comment:
div = <Box> component in MUI, please use Box instead of div

Reply:
done